com.unity.shadergraph/Editor/Unity.ShaderGraph.Editor.asmdef

```diff
@@ -18,7 +18,7 @@
     "versionDefines": [
         {
             "name": "com.unity.rendering.hybrid",
-            "expression": "0.5.0",
+            "expression": "0.5.0-preview.4",
```

Comment:
We might need to update this based on which version of hybrid our other changes land in.